core/controllers/editor.py

```diff
@@ -767,6 +767,28 @@ def get(self, exploration_id, escaped_state_name):
         })
 
 
+class FetchIssuesHandler(EditorHandler):
+    """Handler used for retrieving the list of unresolved issues in an
+    exploration.
+    """
+
+    @acl_decorators.can_view_exploration_stats
+    def get(self, exp_id):
+        """Handles GET requests."""
+        exp_issues = stats_services.get_exp_issues_by_id(exp_id)
+        self.render_json(exp_issues.unresolved_issues)
+
+
+class FetchPlaythroughHandler(EditorHandler):
+    """Handler used for retrieving a playthrough."""
+
+    @acl_decorators.open_access
```

Comment:
@seanlip @brianrodri I'm not sure what to do re this decorator. I can't use can_view_exploration_Stats or something similar because all of them require an <exploration_id> to be a part of the URL for the handler. This handler only needs a playthrough_id to be a part of the handler. Can we leave it to open access because this particular playthrough_id can be obtained only from using the FetchIssues controller which in turn requires the proper decorator to be True.

Reply:
No. If you enter the GET URL for this handler in the browser, the json return value can be accessed directly (try it and see). So the rationale stated in your last sentence doesn't work.

It's ok to have an (unused) exploration_id in the URL. All this stuff is scoped to an exploration anyway, so it's not that unnatural.

Also, follow the existing url conventions. E.g. /playthroughdatahandler/<exploration_id>/<playthrough_id> perhaps.

Reply:
@seanlip Ah, if unused exploration_ids are fine, then there are no issues. 

Re the naming, are you suggesting we rename this handler to PlaythroughDataHandler (similar to ExplorationDataHandler)?